_includes/sidebar.html

```diff
@@ -3,10 +3,12 @@
   <li class="sidebar-title">Get started</li>
 
     <ul class="sidebar-items">
-      <li><a href="/setup/">Install</a></li>
-      <li><a href="/getting-started/">Create and run your first app</a></li>
-      <li><a href="/ide-setup/">Setting up the IDE</a></li>
-      <li><a href="/bootstrap-into-dart/">Bootstrap into Dart</a></li>
+      <li><a href="/get-started/">1: Overview</a></li>
+      <li><a href="/get-started/setup/">2: Setup Flutter</a></li>
+      <li><a href="/get-started/setup-editor/">3: Setup editor</a></li>
```

Comment:
Same as above. Grammatically, it should be "Editor Setup" or "Set up Editor". I prefer "Editor Setup".

Reply:
Done